data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2022 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+mission "Remnant: Visit to Paradise 1"
+	name "Take Taely to visit <stopovers>"
+	description "Take Taely to visit Vail, then return to Caelian."
+	minor
+	source
+		government "Remnant"
+	destination "Caelian"
+	stopover "Vail"
+	to offer
+		has "event: remnant: cognizance calm"
+		or
```

Comment:
It's not, actually. The random value must be greater than 30 to trigger normally. With a defer, the random value needs to be greater than 60. So that should be something close to half as likely to trigger.